apisix/init.lua

```diff
@@ -299,6 +300,8 @@ function _M.http_access_phase()
 
     core.ctx.set_vars_meta(api_ctx)
 
+    core.response.set_header("server", ver_header)
```

Comment:
I would recommend adding a more independent header to describe the version of APISIX, e.g {"X-Version": "1.4"}

Reply:
The current way is more common, we should use the same way.

* tengine: `Server: Tengine/2.2.3`
* kong: `Server: kong/1.4.0`
* nginx: `Server: nginx/1.14.2`

Reply:
Agreed. Moreover `X-Version : 1.4` does not convey any information related to `apisix`